vizro-core/src/vizro/models/_components/form/slider.py

```diff
@@ -58,9 +60,13 @@ class Slider(VizroBaseModel):
     _set_default_marks = validator("marks", allow_reuse=True, always=True)(set_default_marks)
     _set_actions = _action_validator_factory("value")
 
-    @_log_call
-    def build(self):
-        init_value = self.value or self.min
+    def __call__(self, current_value=None, new_min=None, new_max=None, **kwargs):
+        return self._build_static(current_value=current_value, new_min=new_min, new_max=new_max, **kwargs)
+
+    def _build_static(self, current_value=None, new_min=None, new_max=None, **kwargs):
+        _min = new_min if new_min else self.min
+        _max = new_max if new_max else self.max
+        init_value = current_value or self.value or _min
```

Comment:
Can you remind me what's happening here? Why do we need `current_value` here and not just the min and max?

Also, following comment elsewhere let's do something like this unless there's good reason to not do so:

```suggestion
    def _build_static(self, value, min, max, **kwargs):
        init_value = value or min
```

Reply:
ok great, thanks! I'm starting to lose track of what gets simplified here when we fix the Dash persistence bug. So long as we fix that next and you remember it's all good though 😀